packages/vscode/languages/astro-language-configuration.json

```diff
@@ -5,8 +5,7 @@
   "brackets": [
     ["<!--", "-->"],
     ["<", ">"],
-    ["{", "}"],
-    ["(", ")"]
```

Comment:
There's no cases where you want parenthesizes to count as brackets in the `source.astro` scope, in context where they do matter (ex: `source.ts`, `source.js`, `source.tsx`) the languages used there will add them as brackets